projects/packages/forms/src/dashboard/about/index.tsx

```diff
@@ -0,0 +1,235 @@
+					'jetpack-forms'
+				),
+			},
+			{
+				image: `${ ASSETS_URL }/images/appointment-form.png`,
+				title: __( 'Appointment form', 'jetpack-forms' ),
+				description: __(
+					'Let users schedule calls, consultations or meetings with ease.',
+					'jetpack-forms'
+				),
+			},
+			{
+				image: `${ ASSETS_URL }/images/salesforce-form.png`,
+				title: __( 'Salesforce lead form', 'jetpack-forms' ),
+				description: __( 'Generate and sync leads directly to Salesforce CRM.', 'jetpack-forms' ),
+			},
+		],
+		[ ASSETS_URL ]
+	);
+
+	const isWpcomSite = isWpcomPlatformSite();
+
+	return (
+		<div className="jp-forms__about">
+			<div className="section-patterns">
+				<div className="section-patterns__header">
+					<h1>{ __( 'The simplest way to create forms', 'jetpack-forms' ) }</h1>
+					<p className="section-patterns__header-description">
+						{ __( 'Start with one of many patterns, customize to your needs', 'jetpack-forms' ) }
+					</p>
+					<CreateFormButton showPatterns />
+				</div>
+				<div className="section-patterns__grid">
+					{ patterns.map( pattern => (
+						<Card key={ pattern.title } className="section-patterns__grid-card">
+							<CardBody>
+								<div className="section-patterns__grid-card-body-wrapper">
+									<img src={ pattern.image } alt={ pattern.title } />
+								</div>
+							</CardBody>
+							<CardFooter>
+								<div className="section-patterns__grid-card-footer">
+									<div className="section-patterns__grid-card-title">
+										<h4>{ pattern.title }</h4>
+										{ pattern.recommended && (
+											<div>
+												<span className="section-patterns__grid-card-recommended-badge">
+													<Dashicon icon="yes-alt" size={ 16 } />
+													{ __( 'Recommended', 'jetpack-forms' ) }
+												</span>
+											</div>
+										) }
+									</div>
+									<p>{ pattern.description }</p>
+								</div>
+							</CardFooter>
+						</Card>
+					) ) }
+				</div>
+			</div>
+			<div className="section-data">
+				<div className="section-data__container">
+					<h1>{ __( 'You’re in full control of the data.', 'jetpack-forms' ) }</h1>
+					<h1>{ __( 'Empower your workflow.', 'jetpack-forms' ) }</h1>
+					<div className="section-data__features">
+						<div className="section-data__features-feature feature-connect">
+							<div className="app-icons-wrapper">
+								<img
+									src={ `${ ASSETS_URL }/images/akismet-icon.png` }
+									alt={ __( 'Akismet icon', 'jetpack-forms' ) }
+								/>
+								<img
+									src={ `${ ASSETS_URL }/images/jetpack-icon.png` }
+									alt={ __( 'Jetpack icon', 'jetpack-forms' ) }
+								/>
+								<img
+									src={ `${ ASSETS_URL }/images/google-sheets-icon.png` }
+									alt={ __( 'Google Sheets icon', 'jetpack-forms' ) }
+								/>
+								<img
+									src={ `${ ASSETS_URL }/images/salesforce-icon.png` }
+									alt={ __( 'Salesforce icon', 'jetpack-forms' ) }
+								/>
```

Comment:
As with other question about Akisment, I wonder if it would make sense to use existing jetpack svg icon. Example [here](https://github.com/Automattic/jetpack/blob/18e11548102bfeaa1e3e4b4d5d217ba752eacf56/projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/jetpack-crm-card.js#L2). 

I noticed the salesforce png is a bit blurry. I have a separate [PR](https://github.com/Automattic/jetpack/pull/43297/files) that adds a salesforce svg icon. Once merged we could update that here too in a follow up.